easybuild/easyconfigs/d/DL_POLY_Classic/DL_POLY_Classic-1.9-foss-2015b-PLUMED-2.1.4.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'MakeCp'
+
+name = 'DL_POLY_Classic'
+version = '1.9'
+
+homepage = 'http://ccpforge.cse.rl.ac.uk/gf/project/dl_poly_classic/'
+description = """DL_POLY Classic is a freely available molecular dynamics program developed
+ from the DL_POLY_2 package. This version does not install the java gui."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'usempi': True}
+
+sources = ['dl_class_%(version)s.tar.gz']
+source_urls = ['http://ccpforge.cse.rl.ac.uk/gf/download/frsrelease/255/2627/']
+
+plumedversion = '2.1.4'
+versionsuffix = '-PLUMED-%s' % plumedversion
+
+dependencies = [('PLUMED', plumedversion)]
+
+# this patch needs to be regenerated for another version of PLUMED
+patches = [('PLUMED-%s.patch' % plumedversion, 1)]
+
+prebuildopts = 'cd srcmod && '
+buildopts = 'gfortran'
+
+files_to_copy = [(["execute/cleanup", "execute/copy", "execute/DLPOLY.X",
+                   "execute/gopoly", "execute/gui", "execute/select",
+                   "execute/store", "execute/supa"], 'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/DLPOLY.X'],
+    'dirs': []
+}
```

Comment:
bleh, so much stuff that could be handled once-and-forget by updating the `DL_POLY_Classic` easyblock...

I can look into that if you're not up for it